Fuzzlyn/FuzzlynOptions.cs

```diff
@@ -245,6 +246,8 @@ internal class FuzzlynOptions
             [(int)VectorCreationKind.CreateBroadcast] = 0.44,
             [(int)VectorCreationKind.CreateScalar] = 0.44,
         });
+
+    public ProbabilityDistribution SwitchCaseCountDist { get; set; } = new GeometricDistribution(0.2, 5);
```

Comment:
```suggestion
    public ProbabilityDistribution SwitchCaseCountDist { get; set; } = new GeometricDistribution(0.7, 3);
```

Would suggest something like this to make the variance a bit higher

Reply:
What do you mean by "variance" here? With your suggestion, the vast majority of switches will have only 3 case labels (plus default). With a couple sample runs of 100 programs generated, I never see 8 cases, and only a couple 7 cases. It does look (from SharpLab) like Roslyn generates `switch` IL starting with 3 cases, so this would achieve that goal.

Reply:
> What do you mean by "variance" here? With your suggestion, the vast majority of switches will have only 3 case labels (plus default). With a couple sample runs of 100 programs generated, I never see 8 cases, and only a couple 7 cases. It does look (from SharpLab) like Roslyn generates switch IL starting with 3 cases, so this would achieve that goal.

Sorry, I think I got the direction of the probability reversed. I think what you had before is better (to get higher variance)